Moonscraper Chart Editor/Assets/Scripts/Game/Charts/IO/Midi/MidReader.cs

```diff
@@ -719,11 +814,19 @@ static void BuildDrumsMidiNoteNumberToProcessFnDict()
             {
                 int pad = (int)keyVal.Key;
                 int midiKey = keyVal.Value;
-                DrumsMidiNoteNumberToProcessFnMap.Add(midiKey, (in EventProcessParams eventProcessParams) =>
+
+                if (processFnDict.ContainsKey(midiKey))
+                {
+                    processFnDict.Remove(midiKey);
+                }
+
+                processFnDict.Add(midiKey, (in EventProcessParams eventProcessParams) =>
```

Comment:
```suggestion

                processFnDict[midiKey] = (in EventProcessParams eventProcessParams) =>
```

Reply:
I actually need to remove that entirely, that was only necessary for the previous implementation of switching between non-dynamics and dynamics.